CHANGES.md

```diff
@@ -119,6 +119,9 @@ UI:
 -   Fixed CSV loader didn't retry the different line ending correctly
 -   Saved publisher id in the database
 -   Added hover text for the tooltip beside the date-picker in the Add Dataset page
+-   Added a distribution hyperlink to the title of resource links
```

Comment:
When you merge the master, this line has been deplicated

Reply:
Oh good catch!